internal/registry/registry_cache_test.go

```diff
@@ -189,6 +189,10 @@ func testRegistryCache(t *testing.T, when spec.G, it spec.S) {
 				err = registryCache.Refresh()
 				h.AssertError(t, err, "initializing")
 			})
+
+			it.After(func() {
+				h.AssertNil(t, os.RemoveAll(filepath.Join(registryCache.Root, registryCache.RegistryDir)))
```

Comment:
```suggestion
				h.AssertNil(t, os.RemoveAll(registryCache.RegistryDir))
```

I hate to be persnickety, but the `.RegistryDir` is already an absolute path. Having the `.Join` there is just a _tad_ more confusing. Enough for me to mention it.